rear_rider_device/main.py

```diff
@@ -16,12 +16,12 @@ def main():
             leds_child_process=leds_child_proc)
     accelerometer_proc = AccelerometerChildProcess(buf_size=32, fps=1,
         bt_server_proc=bt_server_process)
-    camera_proc = CameraChildProcess(bt_server_proc=bt_server_process)
+    # camera_proc = CameraChildProcess(bt_server_proc=bt_server_process)
     child_processes: list[Process] = [
         leds_child_proc,
         accelerometer_proc,
         bt_server_process,
-        camera_proc
+        # camera_proc
```

Comment:
did you mean to leave these two commented out?

Reply:
Yes, the camera process was bugging out for me, it was related to the fix you committed for it.

Reply:
alright, i'll put the camera fix in a separate PR, also since matt is going to roll with the lidar sensor